lib/snapshot/BUILD.gn

```diff
@@ -63,6 +63,9 @@ group("generate_snapshot_bins") {
 # See: `bin_to_linkable` rules below that build these outputs into linkable form
 # See: https://github.com/flutter/flutter/wiki/Flutter-engine-operation-in-AOT-Mode
 compiled_action("generate_snapshot_bin") {
+  # TODO(https://github.com/flutter/flutter/issues/154437).
+  prefix_with_time_cmd = true
```

Comment:
Does this flag need to be added to https://github.com/flutter/buildroot repo first?

Reply:
Oh shoot, I thought I had sent the buildroot patch, let me send you that first!

Reply:
https://github.com/flutter/buildroot/pull/902

Reply:
Rolled buildroot in this PR as well.